File: pacote-download/Iniciante/JavaScript/Uri1074-ParOuImpar.js
var input = require('fs').readFileSync('JavaScript/stdin', 'utf8');
var lines = input.split('\n')

var valor = Number(lines.shift())
var contador = 0

while (contador < valor) {
    if (lines[contador] == 0) {
        console.log('NULL')
    }

    if (lines[contador] % 2 == 0) {
        if (lines[contador] > 0) {
            console.log('EVEN POSITIVE')
        }
        if (lines[contador] < 0) {
            console.log('EVEN NEGATIVE')
        }
    }

    if (lines[contador] % 2 != 0) {
        if (lines[contador] > 0) {
                console.log('ODD POSITIVE')
        }
        if (lines[contador] < 0) {
                console.log('ODD NEGATIVE')
        }
    }
    contador += 1
}

File: pacote-download/Iniciante/Python/Uri1074-ParOuImpar.py
valor = int(input())
lista = list()
contador = 0

while contador < valor:
    lista.append(int(input()))
    if lista[contador] % 2 == 0:
        if lista[contador] > 0:
            print('EVEN POSITIVE')
        elif lista[contador] < 0:
            print('EVEN NEGATIVE')
    elif lista[contador] % 2 != 0:
        if lista[contador] > 0:
            print('ODD POSITIVE')
        elif lista[contador] < 0:
            print('ODD NEGATIVE')
    if lista[contador] == 0:
        print('NULL')
    contador += 1
